### ISSUE TITLE
WordPress Multisite (MS) support

### ISSUE THREAD

**Issue by slaFFik** [slaFFik/BP-Groups-Extras#70]
https://wordpress.org/support/topic/shifting-to-multisite?replies=1#post-8202048

https://github.com/slaFFik/BP-Groups-Extras/issues/64

**Comment by slaFFik** [comment]
https://wordpress.org/support/topic/groups-extras-link-in-the-admin-settings-is-gone?replies=3

**Comment by slaFFik** [comment]
https://wordpress.org/support/topic/buddypress-in-multi-site?replies=1

**Comment by tammy1999** [comment]
It does work with a multi site because we are using it now on a multi site.  But we didn't set that website up.

**Comment by robmcclel** [comment]
Currently, when Network activated, it has a Group Pages item added to the menu of every site in the network, though that menu doesn't access anything.

It would be super cool if that menu could allow any Group admin, who is also a site holder, access to their pages.  Given there are a lot more editing capabilities in the back end, it could be really helpful for making more complex pages (easily add graphics, for example).

**Comment by BusinessWing** [comment]
Yes, it will be super good if it can let each site's GROUP OWNER and/or Admin has ability to manage/edit his groups (inc., page, photos, forums) at the site it belongs to. his is what I am looking for.

**Comment by slaFFik** [comment]
Related: https://wordpress.org/support/topic/changes-in-admin-do-not-stick/

**Comment by kalico1** [comment]
Thank you @slaFFik for adding my report to this thread. Is it possible to get around this in some way for now? Can I make my desired changes in the database?

**Comment by slaFFik** [comment]
Related: https://wordpress.org/support/topic/can-it-work-in-multi-site/#post-8476564

**Comment by slaFFik** [comment]
https://wordpress.org/support/topic/fundamentally-broken-on-subsites/
